Plot.R

```diff
@@ -113,241 +137,731 @@ parsesubject <- function(row, half){
+dev.off()
+
+# HCR008
+HCR008.set2 <- subset(subset(Full.with.energy, Subject == "HCR008"), Days >=55)
+HCR008.set2["Subject"] <- NULL
+postscript("HCR008_Variance_visit2.eps",  width = 480, height = 480)
+PlotR2s(HCR008.set2[, 5:(dim(HCR008.set2)[2] - 1)],
+        "Variance Explained in HCR008 (visit 2)", "energy")
+dev.off()
+
+# HCR009
+HCR009.set2 <- subset(subset(Full.with.energy, Subject == "HCR009"), Days >= 62)
+HCR009.set2["Subject"] <- NULL
+postscript("HCR009_Variance_visit2.eps",  width = 480, height = 480)
+PlotR2s(HCR009.set2[, 5:(dim(HCR009.set2)[2]-1)],
+        "Variance Explained in HCR009 (visit 2)", "energy")
+dev.off()
+
+
+
+
 ################################
 # Communication and activity PCA
 ################################
 
 communication.activity <- (read.csv("communication.activity.csv"))
-communication.activity['subject'] <- (apply(communication.activity, 1, parsesubject, 2))
+communication.activity['subject'] <- 
+  (apply(communication.activity, 1, parsesubject, 2))
 
 TimeSubj <- data.frame(communication.activity$TimeSubjectIndex)
 subj <- data.frame(communication.activity$subject)
 length <- as.numeric(dim(communication.activity)[2])
 Matrix_For_PCA <- communication.activity[,3:(length-1)]
-Act_Com_PCA <- prcomp(Matrix_For_PCA, scale.=TRUE)
-ActCom_PCA_Matrix <- data.frame(Act_Com_PCA$x)
+act.com.PCA <- prcomp(Matrix_For_PCA, scale.= TRUE)
+act.com.pca.matrix <- data.frame(act.com.PCA$x)
 
-ActCom_PCA_Matrix["Subject"] <- subj
-write.table(Act_Com_PCA$rotation, "Act_Com_Loadings.csv", sep=",")
+act.com.pca.matrix["Subject"] <- subj
+write.table(act.com.PCA$rotation, "Act_Com_Loadings.csv", sep = ",")
 
 
-ActCom_PCA_Matrix["TimeSubjectIndex"] <- TimeSubj
+act.com.pca.matrix["TimeSubjectIndex"] <- TimeSubj
   
-ActCom_PCA_Matrix["TimesOnly"] <- apply(ActCom_PCA_Matrix, 1, parsesubject, 1)
+act.com.pca.matrix["TimesOnly"] <- 
+  apply(act.com.pca.matrix, 1, parsesubject, 1)
 
-windowed_ActCom_PCA <- subset(ActCom_PCA_Matrix, TimesOnly %in% TimeList)
+windowed_ActCom_PCA <- 
+  subset(act.com.pca.matrix, TimesOnly %in% TimeList)
 
 ###########
 # PCA Plots
 ###########
-colorblind_Palette <- c("#000000", "#0072B2", "#56B4E9", "#F0E442", "#D55E00", "#CC79A7")
-
-postscript("Act_Com_PCAPlotAll.eps", width=480, height=480)
-p1 <- qplot(x=PC2, y=PC3, data=ActCom_PCA_Matrix, colour=Subject) + 
-            scale_colour_manual(values=colorblind_Palette)
-p2 <- qplot(x=PC1, y=PC3, data=ActCom_PCA_Matrix, colour=Subject) + 
-            scale_colour_manual(values=colorblind_Palette)
-p3 <- qplot(x=PC1, y=PC2, data=ActCom_PCA_Matrix, colour=Subject) + 
-            scale_colour_manual(values=colorblind_Palette)
-grid.arrange(p1, p2, p3, ncol=3, 
-             top="Principal Components for Combined Activity and Communication (All Measurements)")
+colorblind_Palette <- c("#000000", "#0072B2", "#56B4E9", "#F0E442",
+                        "#D55E00", "#CC79A7")
+
+postscript("act.com.PCA.PlotAll.eps", width=480, height=480)
```

Comment:
so many pca plots!! Would it be easy to make this into a function?

One additional benefit of making them functions is if one thing needs to be tweaked in all plots, you'll only need to change the one function instead of having to worry about making the change a bunch of times